stockflux-chart/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^3.3.6",
+    "BitFlux": "github:scottlogic/BitFlux#1.8.1",
+    "grunt-cli": "^1.2.0",
```

Comment:
this should probably be a dev dependency

Reply:
What is this used for, can we remove it. Feels weird using grunt when we've moved to react scripts?

Reply:
Also can we add a task to the shoet term backlog to moved styles in to the common components module which would includr our dep on bootstrap.

Reply:
It's used to build BitFlux post install